service/src/main/resources/application.yml

```diff
@@ -0,0 +1,55 @@
+# All env variables that are used in one place
+# This is for deployment-specific values, which may be managed by other teams
+env:
```

Comment:
These are not necessary, but do not cause harm.
Every item in this file can be overridden with an envvar. For example, AXONSERVER_SAM_BASEPATH would override the Sam base path without needing this extra twiddle.